temba/api/v2/views.py

```diff
@@ -3573,3 +3581,454 @@ def post(self, request, *args, **kwargs):
+
+        Create new collection for current org:
+
+            POST /api/v2/database.json
+            {
+                "collection_name": "Lookups List"
+            }
+
+
+        Response status of successful creation would be `201`
+
+        ## Uploading data to Lookup Collection
+
+        A `PUT` inserts data into collection:
+
+         * **collection_name** - the name of collection
+         * **fields** - columns that are going to be created in collection (If not provided the columns will be created automaticaly from items keywords)
+         * **items** - list of objects that are going to be inserted
+
+        Create new collection for current org (This action will clear all records and paste new ones instead of them):
+
+            PUT /api/v2/database.json
+            {
+                "collection_name": "Lookups List",
+                "fields": {
+                    "name": {"type": "String"},
+                    "age": {"type": "Number"}
+                },
+                "items": [
+                    {
+                        "name": "Test Name",
+                        "age": 50
+                    },
+                    {
+                        "name": "Test Name 2",
+                        "age": 22
+                    }
+                ]
+            }
+
+
+        Response will contain list of created objects and response status will be `201`:
+
+            [
+                {
+                    "success": {
+                        "objectId": "QnzUnlfydT",
+                        "createdAt": "2020-12-10T15:51:12.797Z"
+                    }
+                },
+                {
+                    "success": {
+                        "objectId": "uJTNjXyek9",
+                        "createdAt": "2020-12-10T15:51:12.797Z"
+                    }
+                }
+            ]
+
+        ## Deleting Lookup Collection
+
+        A `DELETE` deletes collection:
+
+         * **collection_name** - the name of collection
+
+        Create new collection for current org:
+
+            DELETE /api/v2/database.json
+            {
+                "collection_name": "Lookups List"
+            }
+
+
+        Response status of successful deletion would be `204`
+    """
+
+    permission = "orgs.org_lookups"
+    parse_headers = {
+        "X-Parse-Application-Id": settings.PARSE_APP_ID,
+        "X-Parse-Master-Key": settings.PARSE_MASTER_KEY,
+        "Content-Type": "application/json",
+    }
+
+    @staticmethod
+    def get_collection_full_name(org, collection, collection_type=LOOKUPS.lower()):
+        slug_new_collection = slugify(collection)
+        collection_full_name = (
+            f"{settings.PARSE_SERVER_NAME}_{org.slug}_{org.id}_{collection_type}_{slug_new_collection}"
+        )
+        collection_full_name = collection_full_name.replace("-", "")
+        return collection_full_name
+
+    def get_default_params(self, config=""):
```

Comment:
I think we can make the variable `config` as a `boolean`, and call it something like `is_new_collection`.

Reply:
Replaced config with `is_new_collection` and `is_collection_exists`